cfblog/signals.py

```diff
@@ -0,0 +1,10 @@
+from django.dispatch import Signal
+
+"""
+    pre_publish_signal should return list of tuples with two elements.
```

Comment:
Any handler of `pre_publish_signal` will return two values.

And the results of all of these handlers will be available as a list.